.github/workflows/lint.yaml

```diff
@@ -0,0 +1,26 @@
+name: go lint
+on: push
+jobs:
```

Comment:
So I chose github actions just because the feature set is a little different. I'm still fine using cloudbuild to release and build. 

Reason I chose actions is for arbitrary matrix builds, chronjob triggers without needing to manage terraform, custom badges and  there isn't a reason to add extra access control.